scolapasta-hex/src/tests/mod.rs

```diff
@@ -61,28 +61,28 @@ fn test_literal_exhaustive() {
 
 #[test]
 fn test_escape_byte() {
-    // Expected hex escape codes for all possible u8 values
+    // Expected hex escape codes for all possible `u8` values
     #[rustfmt::skip]
-        const EXPECTED: [&str; 256] = [
```

Comment:
this indentation mishap appears to be from a move of this test in a nested `mod tests { }` block to this top-level decl in the `tests/mod.rs` file.